howtos/how-to-make-a-clickable-avatar-in-a-header/README.md

```diff
@@ -17,4 +17,11 @@ To run this tutorial, you will need:
 1. Clone or download the project
 2. Replace `<APP_ID>` in `server.js` with the value found in your [TalkJS dashboard](https://talkjs.com/dashboard/login).
 3. Run `npm install` to install dependencies.
-4. Run `npm start` to start the server.
\ No newline at end of file
+4. Run `npm start` to start the server.
+5. Add an action button to the chat header of your theme:
+   1. Go to the **Themes** tab of the TalkJS dashboard.
+   2. Select to **Edit** your current theme.
+   3. In the list of **Built-in Components**, select **ChatHeader**.
```

Comment:
ChatHeader isn't available on all themes. Maybe worth adding a disclaimer? Specifically the group_chat is missing it. Or bringing it up with Frederik. He was last looking at this and concluded that it wasn't meant to have a chatheader component, but i'm not 100% sure what's going on with it tbh.

Reply:
Thanks, added a disclaimer for now and will see what Frederik says